<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8"/>
	<title>Identification of low- and high-impact hemagglutinin amino acid substitutions that drive antigenic drift of influenza A(H1N1) viruses</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	
	<!-- RSS feed -->
	<link rel="alternate" type="application/rss+xml" title="Robin Green" href="http://robingreen525.github.io/feed.xml">
	
    <!-- Customized Bootstrap + Font Awesome + Solarized -->
    <link href="/css/style1.css" rel="stylesheet" media="screen">

	<!-- Favicon -->
	 <link rel="shortcut icon" href="/images/logo.jpg"/>	

	<!-- Typekit -->
	<script>
	  (function(d) {
		var config = {
		  kitId: 'xeu8jut',
		  scriptTimeout: 3000
		},
		h=d.documentElement,t=setTimeout(function(){h.className=h.className.replace(/\bwf-loading\b/g,"")+" wf-inactive";},config.scriptTimeout),tk=d.createElement("script"),f=false,s=d.getElementsByTagName("script")[0],a;h.className+=" wf-loading";tk.src='//use.typekit.net/'+config.kitId+'.js';tk.async=true;tk.onload=tk.onreadystatechange=function(){a=this.readyState;if(f||a&&a!="complete"&&a!="loaded")return;f=true;clearTimeout(t);try{Typekit.load(config)}catch(e){}};s.parentNode.insertBefore(tk,s)
	  })(document);
	</script>
		

	<!-- jQuery -->
	<script src="/js/jquery.min.js"></script>
	
	<!-- Bootstrap -->
	<script src="/js/transition.js"></script>
	<script src="/js/collapse.js"></script>

</head>

<body>
	
	<div id="header">
		<nav class="navbar navbar-default navbar-static-top" role="navigation">	 
			<div class="container">	
				<div class="navbar-header">
					<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-element" aria-expanded="false">
						<span class="sr-only">Toggle navigation</span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
					</button>
										
					<a class="navbar-brand" href="/">Home</a>
					
				</div>
				<div class="collapse navbar-collapse" id="navbar-collapse-element">
				<ul class="nav navbar-nav navbar-right">
				<li><a href="/">Home</a></li>
				<li><a href="/pubs/">Publications</a></li>
				<li><a href="/cv/">CV</a></li>
        		
				</div>
			</div>
		</nav>	
	</div>
	
	<div class="container">	
		
	<div class="row">
	<div class="col-md-12">
		<div class="media">
			
			<img class="pull-left pad-right media-object" src="/images/papers/harvey-h1n1-antigenic-drift.png">
			
			<div class="media-body titlebox">
				<div class="title media-heading">
					Identification of low- and high-impact hemagglutinin amino acid substitutions that drive antigenic drift of influenza A(H1N1) viruses
				</div>
				<p>
				<div class="smallhead">
					Harvey WT, Benton DJ, Gregory V, Hall JPJ, Daniels RS, Bedford T, Haydon DT, Hay AJ, McCauley JW, Reeve R.
					2016.
					PLoS Pathog. 12: e1005526.
				</div>
			</div>
		</div>
	</div>				
</div>

<div class="bigspacer"></div>

<div class="row">
	<div class="col-md-3">
		<div class="bigspacer"></div>
		<div class="glyphbox note">
					
			<div class="smallhead">
				PDF
			</div>	
			<div class="pad-left note">
				<div class="smallspacer"></div>
				<i class="fa fa-file-text-o fa-fw"></i>
				<a class="off" href="/pdfs/papers/harvey-h1n1-antigenic-drift.pdf">harvey-h1n1-antigenic-drift.pdf</a>
			</div>		
			<div class="bigspacer"></div>							
					
				
			
			<div class="smallhead">
				DOI
			</div>	
			<div class="pad-left note">
				<div class="smallspacer"></div>
				<i class="fa fa-external-link fa-fw"></i>
				<a class="off" href="http://dx.doi.org/10.1371/journal.ppat.1005526">10.1371/journal.ppat.1005526</a>
			</div>		
			<div class="bigspacer"></div>							
							
						
		</div>
	</div>
	<div class="col-md-8">	
		<div class="post">
			<h1>Abstract</h1>

<p>Determining phenotype from genetic data is a fundamental challenge. Identification of emerging antigenic variants among circulating influenza viruses is critical to the vaccine virus selection process, with vaccine effectiveness maximized when constituents are antigenically similar to circulating viruses. Hemagglutination inhibition (HI) assay data are commonly used to assess influenza antigenicity. Here, sequence and 3D structural information of hemagglutinin (HA) glycoproteins were analyzed together with corresponding HI assay data for former seasonal influenza A(H1N1) virus isolates (1997â€“2009) and reference viruses. The models developed identify and quantify the impact of eighteen amino acid substitutions on the antigenicity of HA, two of which were responsible for major transitions in antigenic phenotype. We used reverse genetics to demonstrate the causal effect on antigenicity for a subset of these substitutions. Information on the impact of substitutions allowed us to predict antigenic phenotypes of emerging viruses directly from HA gene sequence data and accuracy was doubled by including all substitutions causing antigenic changes over a model incorporating only the substitutions with the largest impact. The ability to quantify the phenotypic impact of specific amino acid substitutions should help refine emerging techniques that predict the evolution of virus populations from one year to the next, leading to stronger theoretical foundations for selection of candidate vaccine viruses. These techniques have great potential to be extended to other antigenically variable pathogens.</p>

		</div>	
	</div>			
	<div class="col-md-1"></div>
</div>	
	
	</div>
	
<footer>
    			
</body>
</html>

