<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"/><title>bedford lab / blog</title><meta name="viewport" content="width=device-width, initial-scale=1.0"><link rel="alternate" type="application/rss+xml" title="Robin Green" href="http://robingreen525.github.io/feed.xml"><link href="/css/style.css" rel="stylesheet" media="screen"><link rel="shortcut icon" href="/images/favicon.png"/> <script>!function(e){var t,a={kitId:"xeu8jut",scriptTimeout:3e3},c=e.documentElement,i=setTimeout(function(){c.className=c.className.replace(/\bwf-loading\b/g,"")+" wf-inactive"},a.scriptTimeout),n=e.createElement("script"),o=!1,s=e.getElementsByTagName("script")[0];c.className+=" wf-loading",n.src="//use.typekit.net/"+a.kitId+".js",n.async=!0,n.onload=n.onreadystatechange=function(){if(t=this.readyState,!(o||t&&"complete"!=t&&"loaded"!=t)){o=!0,clearTimeout(i);try{Typekit.load(a)}catch(e){}}},s.parentNode.insertBefore(n,s)}(document)</script> <script>!function(e,a,n,t,c,o,s){e.GoogleAnalyticsObject=c,e[c]=e[c]||function(){(e[c].q=e[c].q||[]).push(arguments)},e[c].l=1*new Date,o=a.createElement(n),s=a.getElementsByTagName(n)[0],o.async=1,o.src=t,s.parentNode.insertBefore(o,s)}(window,document,"script","//www.google-analytics.com/analytics.js","ga"),ga("create","UA-26244371-2","bedford.io"),ga("send","pageview")</script> <script src="/js/jquery.min.js"></script> <script src="/js/transition.js"></script><script src="/js/collapse.js"></script></head><body><div id="header"> <nav class="navbar navbar-default navbar-static-top" role="navigation"><div class="container"><div class="navbar-header"> <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-element" aria-expanded="false"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button> <object class="logo navbar-brand" data="/images/logo.svg" type="image/svg+xml"></object> <a class="navbar-brand" href="/">bedford lab</a></div><div class="collapse navbar-collapse" id="navbar-collapse-element"><ul class="nav navbar-nav navbar-right"><li><a href="/cv/">CV</a></li><li><a href="https://www.linkedin.com/in/robin-green-166b8216?trk=hp-identity-photo">Linkedin</a></li><li><a href="https://scholar.google.com/citations?hl=en&user=AlJH6PoAAAAJ&view_op=list_works&gmla=AJsN-F7JV6S_gSJ_94XZh-Q-fUwKfW3u_mBFHEbgyFtGGViXYAoET_2vVEH74hElcir8X6DZAzjAHWTxsF4eUtnNuGme2K0UIdqMoikbthqErp8PwbsEGaiY00OYrWO31xOZARZI3eyv">Google Scholar</a></li><li><a href="http://www.ncbi.nlm.nih.gov/myncbi/browse/collection/44054066/?sort=date&direction=ascending">NCBI Bibliography</a></li></ul></div></div> </nav></div><div class="container"><div class="row"><div class="col-md-12"><div class="centered-pills"><ul class="nav nav-pills note"><li role="presentation" class="active"> <a class="off" href="/blog/"> <i class="fa fa-refresh fa-fw"></i> Timeline </a></li><li role="presentation"> <a class="off" href="/blog/archive/"> <i class="fa fa-list-ul fa-fw"></i> Index </a></li></ul></div><div class="bigspacer"></div></div></div><div class="post-list"><div class="blog-post"><div class="row"><div class="col-md-1"></div><div class="col-md-10"><div class="media"><div class="media-body"><div class="title media-heading"> <a href="/blog/programmer-real-time-analysis/" class="off">Programmer for real-time phylodynamic analysis</a></div><p><div class="smallhead"> 4 Apr 2016 by <a class="off" href="/team/trevor-bedford/">trvrb</a></div></div></div></div><div class="col-md-1"></div></div><div class="bigspacer"></div><div class="row"><div class="col-md-1"></div><div class="col-md-10"><div class="post"><p>We're looking for a programmer to help with the current push towards real-time analysis of virus evolution. The advertisement follows:</p><blockquote><p>A programmer position is available immediately in the Bedford lab at the Fred Hutch to develop inference algorithms and interactive visualizations of viral outbreaks using DNA/RNA sequence data.</p><p>There has been remarkable progress in <a href="https://en.wikipedia.org/wiki/Viral_phylodynamics">phylodynamic</a> methods which use viral genetic sequence data to infer patterns of epidemic growth and geographic spread as well as patterns of adaptive evolution and strain turnover. However, until recently, these methods were solely applied in retrospective analyses to understand past events. With increasing availability and timeliness of sequence data, it is now becoming possible to perform and share phylodynamic analyses in near real-time.</p><p>Our group is leading this transition to real-time analysis and prediction. Current efforts by the lab include analyses of influenza (<a href="http://nextflu.org">nextflu.org</a>), Ebola (<a href="http://ebola.nextstrain.org">ebola.nextstrain.org</a>) and Zika (<a href="http://nextstrain.org/zika">nextstrain.org/zika/</a>) viruses. These sites are already being used by the Centers for Disease Control and the World Health Organization, particularly for influenza vaccine strain selection, and we believe further development will lead to substantial public health benefit. This project has grown to the point where we need a full-time programmer to generalize, refactor, and maintain our platform for data ingestion, processing and visualization.</p><p>The ideal candidate would have experience in Python (informatic processing pipeline <a href="https://github.com/blab/nextflu/tree/master/augur">augur</a> is written in Python) and Javascript (browser-based visualization <a href="https://github.com/blab/nextflu/tree/master/auspice">auspice</a> is written in Javascript). Experience with databases (we are using <a href="http://www.rethinkdb.com/">rethinkdb</a> for sequence organization) or web application programming would be a plus. The new team member would contribute to refactor the current codebase to make a more general and extensible platform for informatic processing and also to speed up the auspice visualization by reactive JSON pull-downs. Additional development would focus on extending pipelines to other viruses / pathogens and building new features for current viruses, as well as, implementing a robust <a href="https://www.docker.com/">Docker</a>-based deployment pipeline.</p><p>The Fred Hutch is located in South Lake Union in Seattle, WA and offers a dynamic work environment with cutting-edge science and computational resources. The position is available immediately with flexible starting dates. Informal inquires are welcome. Applications will be accepted until the position is filled. We offer a competitive salary commensurate with skills and experience, along with benefits. The Fred Hutch and the Bedford lab are committed to improving diversity in the computational sciences. Applicants of diverse backgrounds are particularly encouraged to apply.</p><p>For more information about the lab, please see the our website at <a href="http://bedford.io">bedford.io</a>. To apply for the position please send (1) current resume, (2) code samples or links to published/distributed code and (3) contact information for two references to trevor<span style="display:none">obfuscate</span>@bedford.io.</p></blockquote></div></div><div class="col-md-1"></div></div><div class="bigspacer"></div><div class="spacer"></div></div><div class="blog-post"><div class="row"><div class="col-md-1"></div><div class="col-md-10"><div class="media"><div class="media-body"><div class="title media-heading"> <a href="/blog/feb-2016-nextflu-report/" class="off">Seasonal influenza circulation patterns and future projections</a></div><p><div class="smallhead"> 15 Feb 2016 by <a class="off" href="/team/trevor-bedford/">trvrb</a></div></div></div></div><div class="col-md-1"></div></div><div class="bigspacer"></div><div class="row"><div class="col-md-1"></div><div class="col-md-10"> <img class="offset" src="/images/blog/h3n2_clade_trajectory.png"><div class="post"><p><a href="https://neherlab.wordpress.com/">Richard Neher</a> and I have compiled a <a href="http://nextflu.org/reports/feb-2016/">report on recent patterns of seasonal influenza virus evolution</a> with projections for future clade behavior based on the <a href="http://nextflu.org">nextflu</a> platform. This is designed to be looking forward to the 2016-2017 flu season. Lots of interesting behavior in H3N2 and H1N1pdm viruses. Within H3N2, the clade 3c2.a has continued to predominate throughout the last year and now genetic diversity is accumulating within 3c2.a. The issue is now watching for new variants within 3c2.a that may be antigenically distinct and thus spread through the virus population. H1N1pdm viruses have seen the rapid rise a novel clade denoted 6b.2 during 2015. This clade is demarcated by the substitutions 84N/162N/216T. The joint substitutions 162N/216T have risen especially rapidly with this season's Northern Hemisphere epidemic comprised of almost entirely 6b.2 viruses. However, detailed antigenic characterization is necessary to determine whether the seasonal influenza vaccine warrants updating.</p></div></div><div class="col-md-1"></div></div><div class="bigspacer"></div><div class="spacer"></div></div><div class="blog-post"><div class="row"><div class="col-md-1"></div><div class="col-md-10"><div class="media"><div class="media-body"><div class="title media-heading"> <a href="/blog/rtss-sieve/" class="off">Strain-specific vaccine efficacy in the RTS,S malaria vaccine</a></div><p><div class="smallhead"> 26 Oct 2015 by <a class="off" href="/team/trevor-bedford/">trvrb</a></div></div></div></div><div class="col-md-1"></div></div><div class="bigspacer"></div><div class="row"><div class="col-md-1"></div><div class="col-md-10"> <img class="offset" src="/images/blog/rtss_sieve.png"><div class="post"><p>In collaboration with <a href="https://www.broadinstitute.org/bios/daniel-neafsey">Dan Neafsey</a> at the Broad Institute, <a href="http://www.hsph.harvard.edu/dyann-wirth/">Dyann Wirth</a> at Harvard, <a href="https://www.fredhutch.org/en/labs/profiles/gilbert-peter.html">Peter Gilbert</a> and <a href="https://www.fredhutch.org/en/labs/profiles/juraska-michal.html">Michal Juraska</a> at the Fred Hutch and a large team of researchers, we've just published <a href="/papers/neafsey-rtss-sieve/">a paper</a> in the <a href="http://dx.doi.org/10.1056/NEJMoa1505819">New England Journal of Medicine</a> showing strain-specific vaccine efficacy in a recent malaria vaccine trial. We found that the <a href="https://en.wikipedia.org/wiki/RTS,S">RTS,S malaria vaccine</a> worked better against strains that were genetically matched to the vaccine antigen compared to unmatched strains. The right hand figure shows that across study sites, parasites matched to the vaccine strain (3D7) are overrepresented among the control group, indicating that the vaccine better protected against infection by 3D7 than against other strains. Proper statistical analysis shows that there was a 1-year vaccine efficacy of 50% against strains perfectly matched to the vaccine antigen and a 1-year vaccine efficacy of 33% against unmatched strains. The vaccine strain 3D7 is at low (~10%) frequency in Sub-Saharan Africa. This suggests that vaccine efficacy could be straight-forwardly improved by just swapping 3D7 for a more common haplotype.</p><p>It's interesting to see strain-specific vaccine for malaria, suggesting that like flu, a good vaccine requires matching to the circulating pathogen strains. This was a fun study to participate in. Hopefully, we could see further improvements to the RTS,S vaccine. Even at current efficacy levels, RTS,S is estimated to have a cost effectiveness of <a href="http://www.malariajournal.com/content/13/1/66">~$150 per disability-adjusted life year</a> saved.</p></div></div><div class="col-md-1"></div></div><div class="bigspacer"></div><div class="spacer"></div></div><div class="blog-post"><div class="row"><div class="col-md-1"></div><div class="col-md-10"><div class="media"><div class="media-body"><div class="title media-heading"> <a href="/blog/visualizing-antigenic-evolution/" class="off">Analysis and visualization of influenza antigenic evolution</a></div><p><div class="smallhead"> 6 Oct 2015 by <a class="off" href="/team/trevor-bedford/">trvrb</a></div></div></div></div><div class="col-md-1"></div></div><div class="bigspacer"></div><div class="row"><div class="col-md-1"></div><div class="col-md-10"> <img class="offset" src="/images/blog/titer_tree.png"><div class="post"><p><a href="https://neherlab.wordpress.com/">Richard Neher</a> and <a href="http://www.kitp.ucsb.edu/shraiman/">Boris Shraiman</a>, along with myself, <a href="http://www.vet.cam.ac.uk/directory/russell">Colin Russell</a> and <a href="http://www.crick.ac.uk/research/worldwide-influenza-centre/">Rod Daniels</a> have just completed a new analysis of antigenic drift in the seasonal influenza virus. We've put a manuscript on <a href="/papers/neher-titers/">"Prediction, dynamics, and visualization of antigenic phenotypes of seasonal influenza viruses"</a> up on the <a href="http://arxiv.org/abs/1510.01195">arXiv</a> and have created an interactive visualization of our model at <a href="http://HI.nextflu.org">HI.nextflu.org</a>.</p><p>Much previous work on this topic, <a href="/papers/bedford-flux/">including my own</a>, has focused on "antigenic cartography" in which antigenic phenotype is embedded into a 2D antigenic map where distances between viruses are proportional to drop in titer in the hemagglutination inhibition (HI) assay. Here, we take a complementary approach where drop in titer is directly mapped to the influenza phylogeny; specific branches are assigned drops in titer. Thus, rather than embedding into 2D Euclidean space, this model assumes embedding into tree space. As a model, it works quite well, allowing prediction of unmeasured titers at high accuracy. In the manuscript, we investigate the correlation between antigenic advancement and clade success, finding that more advanced clades tend to win in the global competition among viruses. However, we observe significant noise when the new clade is at very low frequency.</p><p>A major feature of the cartography-based approach is providing a single pictorial view of the antigenic relationships among viruses. This is enabled by the 2D basis for cartography. The tree-based model that we present here does not permit a single viewpoint. Instead, we provide an <a href="http://HI.nextflu.org">interactive visualization</a> of the model, in which users can click on a particular reference virus in the phylogeny and see how other viruses relate to this focal virus. We allow coloring of the phylogeny by raw HI titer, as well as by model expectations, allowing exploration of <em>why</em> the model behaves as it does. We hope this approach will be useful in investigation of antigenic relationships among circulating influenza viruses.</p></div></div><div class="col-md-1"></div></div><div class="bigspacer"></div><div class="spacer"></div></div><div class="blog-post"><div class="row"><div class="col-md-1"></div><div class="col-md-10"><div class="media"><div class="media-body"><div class="title media-heading"> <a href="/blog/sep-2015-nextflu-report/" class="off">Seasonal influenza circulation patterns and future projections</a></div><p><div class="smallhead"> 24 Sep 2015 by <a class="off" href="/team/trevor-bedford/">trvrb</a></div></div></div></div><div class="col-md-1"></div></div><div class="bigspacer"></div><div class="row"><div class="col-md-1"></div><div class="col-md-10"><div class="post"><p><a href="https://neherlab.wordpress.com/">Richard Neher</a> and I have compiled a <a href="http://nextflu.org/reports/sep-2015/">report on recent patterns of seasonal influenza virus evolution</a> with projections for future clade behavior. The <a href="http://nextflu.org">nextflu</a> site made putting this together really straightforward. Lots of interesting stuff going on. The 3c2.a clade is taking over H3N2 viruses, there's a new clade of H1N1pdm viruses localized in South Asia and B viruses continue to oscillate between competing clades without permanent winners.</p></div></div><div class="col-md-1"></div></div><div class="bigspacer"></div><div class="spacer"></div></div><div class="blog-post"><div class="row"><div class="col-md-1"></div><div class="col-md-10"><div class="media"><div class="media-body"><div class="title media-heading"> <a href="/blog/sismid-course/" class="off">Pathogen evolution, selection and immunity</a></div><p><div class="smallhead"> 20 Jul 2015 by <a class="off" href="/team/trevor-bedford/">trvrb</a></div></div></div></div><div class="col-md-1"></div></div><div class="bigspacer"></div><div class="row"><div class="col-md-1"></div><div class="col-md-10"> <img class="offset" src="/images/blog/wright_fisher.png"><div class="post"><p>This week, <a href="http://cobeylab.uchicago.edu/">Sarah Cobey</a> and I are co-teaching a short course on "Pathogen evolution, selection and immunity" for the 2015 <a href="http://sismid.uw.edu">Summer Institute in Statistics and Modeling in Infectious Diseases (SISMID)</a>. This course is designed as an introduction to epidemiological modeling and evolutionary analysis for antigenically variable pathogens. Here, on the modeling side, we cover immune-mediated competition, multistrain mechanistic models and state-space reconstruction, while on the evolution side we cover the coalescent, phylogenetics and natural selection. We also discuss the immune response in terms of serology and B cell dynamics. I'm very excited about co-teaching this course.</p><p>Sarah and I have worked together to put all the course materials online. The course website is <a href="/projects/sismid/">here</a>. We include slide decks using <a href="https://github.com/hakimel/reveal.js/">Reveal.js</a> for individual topics like <a href="/projects/sismid/competition/slides.html">competition</a> and the <a href="/projects/sismid/coalescent/slides.html">coalescent</a>. We also include introductory exercises for topics like <a href="/projects/sismid/phylogenetics/practical/">phylogenetics using BEAST</a> and <a href="/projects/sismid/ssr/exercise/">state-space reconstruction</a>. The goal was to have something students could view alongside lectures, but also have something to refer to after the course was finished and to link out to follow-up material. Overall, I'm really quite pleased with this set up.</p></div></div><div class="col-md-1"></div></div><div class="bigspacer"></div><div class="spacer"></div></div><div class="blog-post"><div class="row"><div class="col-md-1"></div><div class="col-md-10"><div class="media"><div class="media-body"><div class="title media-heading"> <a href="/blog/new-recruits/" class="off">New recruits</a></div><p><div class="smallhead"> 30 Jun 2015 by <a class="off" href="/team/trevor-bedford/">trvrb</a></div></div></div></div><div class="col-md-1"></div></div><div class="bigspacer"></div><div class="row"><div class="col-md-1"></div><div class="col-md-10"><div class="post"><p>The lab is suddenly growing. We've had 4 new people join in the last ~2 weeks. <a href="/team/allison-black/">Allison Black</a> is starting a UW Epidemiology PhD in the fall, <a href="/team/gabe-erion/">Gabe Erion</a> is doing a summer rotation with the group as part of the UW MD/PhD program, <a href="/team/charlton-callender/">Charlton Callender</a> is an undergraduate at the UW doing a summer research internship as part of the Fred Hutch SURP program and <a href="/team/sidney-bell/">Sidney Bell</a> is starting a 1 year UW Epidemiology Master's adjunct to the MCB PhD program. This should be a fun summer with lots of projects to hack on together. I'm very much looking forward to it.</p></div></div><div class="col-md-1"></div></div><div class="bigspacer"></div><div class="spacer"></div></div><div class="blog-post"><div class="row"><div class="col-md-1"></div><div class="col-md-10"><div class="media"><div class="media-body"><div class="title media-heading"> <a href="/blog/scientific-publishing-practices/" class="off">On scientific publishing practices in the face of public health crises</a></div><p><div class="smallhead"> 22 Jun 2015 by <a class="off" href="/team/trevor-bedford/">trvrb</a></div></div></div></div><div class="col-md-1"></div></div><div class="bigspacer"></div><div class="row"><div class="col-md-1"></div><div class="col-md-10"> <img class="offset" src="/images/blog/scientific_manuscript.jpg"><div class="post"><p>After 400 years and the rise of the internet, the manuscript remains sacrosanct to scientific discourse. The system of scientific journals and peer review prioritizes findings to be <em>right</em> rather than <em>timely</em>, which is usually not such a bad thing, but in the case of infectious disease outbreaks, science comes to be at odds with public health efforts. In the recent case of the <a href="https://en.wikipedia.org/wiki/Ebola_virus_epidemic_in_West_Africa">2014-2015 West African Ebola epidemic</a>, there was an incredibly well-done publication early on by <a href="http://sabetilab.org/">Pardis Sabeti</a> and colleagues in September 2014 that <a href="http://dx.doi.org/10.1126/science.1259657">analyzed Ebola genomes sequences from Sierra Leone through June 2014</a>. Sequences from this paper were made available on Genbank in June. However, following this, little additional genomic data appeared, in large part due to difficulties in shipping samples out of West Africa. Because of the lack of data, during the height of the epidemic in fall 2014, we had no idea what was going in terms of the ongoing evolution of the virus.</p><p>Instead, we are just now learning the details of the evolution and spread of the virus during the course of the epidemic. In the last month, we saw papers published by <a href="http://dx.doi.org/10.1038/nature14490">Tong et al.</a> detailing 175 new Ebola genomes mostly from Western Sierra Leone, <a href="http://dx.doi.org/10.1038/nature14594">Carroll et al.</a> detailing 179 new genomes mostly from Guinea and <a href="http://dx.doi.org/10.1016/j.cell.2015.06.007">Park et al.</a> detailing 232 new genomes mostly from Eastern Sierra Leone. As discussed <a href="/blog/ebola-transmission-dynamics/">previously</a>, I was involved in the evolutionary analyses in the Park et al. paper looking at purifying vs positive selection on the Ebola genome.</p><p>It's unfortunate that it's taken so long for these results to see the full light of day. At a minimum, genome sequences should be released to the community as soon as they're in a cleanly sharable state. In these sorts of analyses, the genome sequence itself isn't so useful or relevant, it's the differences between sequences that gives insight into evolution and transmission. We need to pool data to really understand what's going on. In this case, the Tong et al. paper was first received by Nature on Jan 30, but sequences didn't appear in Genbank nearly until publication on May 13. The Carroll et al. paper was first received on April 9 and sequences were <a href="http://evident-project.eu/newsevents/ebov-sequences/">released publicly</a> on May 11 prior to publication on June 17. This is a nice step forward. Most of the sequences from Park et al. were released as they came off the sequencing machine in <a href="http://virological.org/t/early-release-16-new-ebov-genomes-from-sierra-leone/72">December</a>, <a href="http://virological.org/t/mid-early-release-96-ebov-genomes-from-sierra-leone/96">January</a> and <a href="http://virological.org/t/mid-early-release-152-ebov-genomes-from-sierra-leone/110">March</a> prior to publication on June 18. Additionally, 85 genomes from Guinea were <a href="http://www.pasteur.fr/en/ebov-sequences-guinea">released in pre-publication form</a> by the Institute Pasteur on May 26.</p><p>I see early data release as an important step forward for the community. Tying data release to publication seems like an obviously poor idea when there are public health implications (for more discussion see <a href="http://www.nature.com/news/data-sharing-make-outbreak-research-open-access-1.16966">Yozwiak et al.</a>). However, on Friday, I was quoted by the NYT as saying: <a href="http://www.nytimes.com/2015/06/20/world/africa/genome-studies-show-how-ebola-spread-initially.html">"you could imagine a situation where you don’t really have to publish your Nature paper; instead, you make a blog post"</a>. Here, I'm picturing something more than data sharing. Posting sequences is great. But it still means that insights are locked away prior to publication. We need to move away from the idea of a definitive published manuscript as being the only worthwhile target for scientific outputs. <a href="http://tree.bio.ed.ac.uk/">Andrew Rambaut</a> has set up the public forum <a href="http://virological.org/">virological.org</a> that is now being used to post both data and analyses of ongoing <a href="http://virological.org/t/direct-deep-sequencing-in-sierra-leone-yields-73-new-ebov-genomes-from-february-may-2015/134">Ebola</a> and <a href="http://virological.org/t/preliminary-analysis-of-middle-east-respiratory-syndrome-coronavirus-mers-cov-sequences-from-korea-and-china/143">MERS-CoV</a> evolution in near real-time by <a href="https://www.sanger.ac.uk/research/faculty/pkellam/">Paul Kellam</a> and colleagues. Data and analyses can be shared publicly online before publication. The paper can come later and represent the definitive analysis. I recognize that papers still need to be produced to fill out CVs and to cement the scientific literature, but a lot can be done before the paper appears. Preprints only get us part of the way there (although <a href="http://haldanessieve.org/">Haldane's Sieve</a> is a fantastic idea). They usually appear alongside initial manuscript submission in fairly polished form. We need lighter weight discourse as well.</p><p>Previously, I blogged about the need for a <a href="/blog/github-of-science/">"GitHub of Science"</a>. I now believe that plain old GitHub is the "GitHub of Science". Scripts and pipelines can be posted to GitHub; results and analyses can be discussed on public forums (like <a href="http://virological.org/">virological.org</a>); figures posted to <a href="http://figshare.com/">figshare</a>; papers can be reviewed after publication via blog. There's no technological limitation to making this work. We have all the tools we need at our disposal. Let's commit to doing a better job of this. In some cases, actual lives are at stake.</p></div></div><div class="col-md-1"></div></div><div class="bigspacer"></div><div class="spacer"></div></div></div><div class="infinite-spinner"></div><div class="row"><div class="col-md-12"><div id="scroll" class="centered-pills"><ul class="nav nav-pills note"><li role="presentation"> <a class="off" href="/blog/"> <i class="fa fa-chevron-up fa-fw"></i> Back to top </a></li></ul></div></div></div> <script async src="/js/infinite-scroll.js"></script></div> <footer></body></html>